src/clockkit.cs

```diff
@@ -996,6 +1660,219 @@ interface CLKComplicationTemplateGraphicCircularStackImage {
 
 		[Export ("line2TextProvider", ArgumentSemantic.Copy)]
 		CLKTextProvider Line2TextProvider { get; set; }
+
+		[Watch (7, 0), NoiOS]
+		[Export ("initWithLine1ImageProvider:line2TextProvider:")]
+		IntPtr Constructor (CLKFullColorImageProvider line1ImageProvider, CLKTextProvider line2TextProvider);
+
+		[Watch (7, 0), NoiOS]
+		[Static]
+		[Export ("templateWithLine1ImageProvider:line2TextProvider:")]
+		CLKComplicationTemplateGraphicCircularStackImage Create (CLKFullColorImageProvider line1ImageProvider, CLKTextProvider line2TextProvider);
+	}
+
+	[Watch (7, 0), iOS (14, 0)]
+	[BaseType (typeof (NSObject))]
+	interface CLKWatchFaceLibrary {
+		[Async]
+		[Export ("addWatchFaceAtURL:completionHandler:")]
+		void AddWatchFace (NSUrl fileUrl, Action<NSError> handler);
 	}
-}
 
+	[Watch (7, 0), NoiOS]
+	[BaseType (typeof (CLKComplicationTemplate))]
+	interface CLKComplicationTemplateGraphicRectangularFullImage : NSSecureCoding {
+		[Export ("imageProvider", ArgumentSemantic.Copy)]
+		CLKFullColorImageProvider ImageProvider { get; set; }
+
+		[Export ("initWithImageProvider:")]
+		IntPtr Constructor (CLKFullColorImageProvider imageProvider);
+
+		[Static]
+		[Export ("templateWithImageProvider:")]
+		CLKComplicationTemplateGraphicRectangularFullImage Create (CLKFullColorImageProvider imageProvider);
+	}
+
+	[Abstract] // from docs 'An abstract superclass for all the extra large circular graphic templates.'
```

Comment:
we had problems with `[Abstract]` before, since Apple can return instance of them.
Might be better to have no public ctor (and no member) and an `[Advice]` this is meant to abstract

Reply:
Do we want to deal with the one in https://github.com/xamarin/xamarin-macios/blob/main/src/clockkit.cs#L715 too? That is the main reason I used Abstract here, to follow the same pattern.

Reply:
I wonder if we should be _fixing_ this at the generator level, i.e. `[Abstract]` on a type to mean `[Advice ("blah!")]`

Reply:
> I wonder if we should be _fixing_ this at the generator level, i.e. `[Abstract]` on a type to mean `[Advice ("blah!")]`

It's already filed: https://github.com/xamarin/xamarin-macios/issues/4969